diff --git a/node_modules/vuetify/lib/util/globals.mjs b/node_modules/vuetify/lib/util/globals.mjs
index d31cbce..e69cbee 100644
--- a/node_modules/vuetify/lib/util/globals.mjs
+++ b/node_modules/vuetify/lib/util/globals.mjs
@@ -1,7 +1,7 @@
 export const IS_NODE = typeof process !== 'undefined';
 export const IN_BROWSER = typeof window !== 'undefined';
-export const IS_DEBUG = IS_NODE && process.env.DEBUG === 'true';
-export const IS_PROD = IS_NODE && process.env.NODE_ENV === 'production';
+export const IS_DEBUG = IS_NODE && process?.env?.DEBUG === 'true';
+export const IS_PROD = IS_NODE && process?.env?.NODE_ENV === 'production';
 export const SUPPORTS_INTERSECTION = IN_BROWSER && 'IntersectionObserver' in window;
 export const SUPPORTS_TOUCH = IN_BROWSER && ('ontouchstart' in window || window.navigator.maxTouchPoints > 0);
 //# sourceMappingURL=globals.mjs.map
